DigitalLearningSolutions.Web/ViewModels/TrackingSystem/CourseSetup/RemoveAdminFieldViewModel.cs

```diff
@@ -17,6 +17,6 @@ public RemoveAdminFieldViewModel(int customisationId, string promptName, int del
 
         public bool Confirm { get; set; }
 
-        public int DelegateCount { get; set; }
+        public int AdminCount { get; set; }
```

Comment:
Not sure why this is named like this. Shouldn't it be Delegate?

Reply:
Oh right, I changed the method GetAdminCountWithAnswerForCourseAdminField to GetDelegateCountWithAnswerForCourseAdminField like you said before, but the number it returns is used for the line "___ admins have responded to this field." Are you sure Delegate would be more accurate?

Reply:
I'm not 100% sure. @stellake Is the wording here correct? It matches the ticket desription, but Progress records aren't associated with Admins.

Reply:
Yes, that's correct - admin fields are basically fields that can be answered by supervisors. E.g. "course notes", where the admin can describe how well the delegate is doing on a course etc.

Reply:
Yeah I think AdminCount is confusing, I already suggested updating some of the method names as they didn't really make sense before this got to review, and the wording on the confirmation page is probably also incorrect in that case. I'd suggest something more like "Admins have responded to this field for X delegates"

Reply:
Cool, I've renamed it to AnswerCount, I think that sounds more intuitive.
@stellake Where would be the right place to document this on swiki?

Reply:
I like the wording change suggestion - can we use that & modify the JIRA please? The documentation can be a "Tracking system" subpage on swiki https://softwiretech.atlassian.net/wiki/spaces/HEE/pages/19507118251/Tracking+System

Reply:
updated the JIRA ticket with the new wording, and I'll work on the swiki documentation today